react/components/RenderProvider.tsx

```diff
@@ -525,15 +556,49 @@ class RenderProvider extends Component<Props, RenderProviderState> {
       )
     }
 
-    // Shows a generic preview page when navigating. In the future, the
-    // preview should be according to the entitiy (department, search, product),
-    // and the fallback should be the generic preview.
-    this.setState(
-      {
-        preview: true,
-      },
-      () => this.scrollTo(state.scrollOptions)
-    )
+    if (navigationRoute.id) {
+      let updatedExtensions: Extensions = {}
+
+      if (window.__RUNTIME__.hasNewExtensions) {
+        // TODO: Remove this when new pages-graphql get released
+        updatedExtensions = generateExtensions(
+          // eslint-disable-next-line
+          blocksTree!,
+          // eslint-disable-next-line
+          blocks!,
+          // eslint-disable-next-line
+          contentMap!,
+          pagesState[page]
+        )
+      }
+      this.setState(
+        {
+          extensions: replaceExtensionsWithDefault(
+            { ...updatedExtensions, ...this.state.extensions },
+            page,
+            defaultExtensions
```

Comment:
It's always empty. We don't fetch default extensions, we are just prefetching components.